templates/tutorial.html

```diff
@@ -55,9 +55,34 @@
       Thanks for reading this tutorial! If you have any feedback,
       please join the discussion below, or open issues and pull
       requests
-      on <a href="https://github.com/stackbuilders/tutorials">GitHub</href>.
+      on <a href="https://github.com/stackbuilders/tutorials">GitHub</a>.
     </p>
 
+    <section>
+      <div class="subscribe-tutorials">
+        <div class="subscribe-tutorials__container">
+          <div class="subscribe-tutorials__form-section">
+            <form id="subscriber_form" action="" method="POST" _lpchecked="1">
```

Comment:
Could you remove `_lpchecked="1`, please?
Also, the `action` will send a POST to `https://stackbuilders.com/tutorials/haskell/generics`. Please specify the proper endpoint